src/pinns_pde_solve.jl

```diff
@@ -793,12 +860,13 @@ function SciMLBase.symbolic_discretize(pde_system::PDESystem, discretization::Ph
     phi = discretization.phi
     derivative = discretization.derivative
     strategy = discretization.strategy
+    global integral = get_numeric_integral(strategy, pde_system.indvars, pde_system.depvars, dict_indvars, chain, derivative)
```

Comment:
global is bad. it will always lead to trouble. Avoid it

Reply:
> in this case I needed to use integral outside the `symbolic_discretize` and `discretize`. And hence global declaration. What are other ways to do it?